dbt_style_guide.md

```diff
@@ -103,19 +164,71 @@ select * from filtered_events
 ## SQL style guide
 
 - Use trailing commas
-- Indents should be four spaces (except for predicates, which should line up with the `where` keyword)
-- Lines of SQL should be no longer than 80 characters
+
+- Indents should be two spaces (except for predicates, which should line up with the `where` keyword)
+
+- Lines of SQL should be no longer than 80 characters and new lines should be used to ensure this.  
+  Example:
+  ```sql
+  sum(case
+    when order_status = 'complete'
+    then order_total
+  end) as monthly_total,
+
+
+  {{ get_windowed_values(
+        strategy='sum',
+        partition='order_id',
+        order_by='created_at',
+        column_list=[
+          'final_cost'
+        ]
+  ) }} as total_final_cost
+  ```
+
 - Field names and function names should all be lowercase
+
 - The `as` keyword should be used when aliasing a field or table
+
 - Fields should be stated before aggregates / window functions
+
 - Aggregations should be executed as early as possible before joining to another table.
+
 - Ordering and grouping by a number (eg. group by 1, 2) is preferred over listing the column names (see [this rant](https://blog.getdbt.com/write-better-sql-a-defense-of-group-by-1/) for why). Note that if you are grouping by more than a few columns, it may be worth revisiting your model design.
+
 - Prefer `union all` to `union` [*](http://docs.aws.amazon.com/redshift/latest/dg/c_example_unionall_query.html)
+
 - Avoid table aliases in join conditions (especially initialisms) – it's harder to understand what the table called "c" is compared to "customers".
+
 - If joining two or more tables, _always_ prefix your column names with the table alias. If only selecting from one table, prefixes are not needed.
+
 - Be explicit about your join (i.e. write `inner join` instead of `join`). `left joins` are normally the most useful, `right joins` often indicate that you should change which table you select `from` and which one you `join` to.
 
-- *DO NOT OPTIMIZE FOR A SMALLER NUMBER OF LINES OF CODE. NEWLINES ARE CHEAP, BRAIN TIME IS EXPENSIVE*
+- When filtering by multiple clauses, each clause and expression  should be on a new line.  
+Example:
+  ```sql
+  where user_id is not null
+  and status = 'pending'
+  and location = 'hq'
+  ```
+
+- Joins should list the "left" table first (i.e., the table you're joining data to):
+```sql
+select
+    
+    trips.*,
+    drivers.rating as driver_rating,
+    riders.rating as rider_rating
+
+from trips
+left join users as drivers
+    on trips.driver_id = drivers.user_id
+left join users as riders
+    on trips.rider_id = riders.user_id
+
+```
+
+- *DO NOT OPTIMIZE FOR A SMALLER NUMBER OF LINES OF CODE. NEWLINES ARE CHEAP, BRAIN TIME IS EXPENSIVE*  

```

Comment:
I would love for this to be in a little "First Principles" section at the top that orients readers to the core philosophy that is guiding this these recommendations. This could easily serve as its core.

Reply:
I accidently committed a working version of this earlier, but it is now at the top of this section. Let me know if there was something else you had in mind! [Commit](https://github.com/dbt-labs/corp/pull/63/commits/b315a601f908da888a9928e2872815873825fd3b)